pkg/util/file_indexer.go

```diff
@@ -106,29 +107,31 @@ func addOdoDirectory(gitIgnoreFile string, fs filesystem.Filesystem) error {
 }
 
 // TouchGitIgnoreFile checks .gitignore file exists or not, if not then create it
-func TouchGitIgnoreFile(directory string) (string, error) {
+func TouchGitIgnoreFile(directory string) (string, bool, error) {
```

Comment:
Could you document what is the significance of the bool returned value. Or I will never remember if it is because it has been created orif it already exists